### ISSUE TITLE
The border colour failed the DAC assessment

### ISSUE THREAD

**Issue by gvonkoss** [Financial-Times/o-share#132]
Border colour for the share icons needs to be darker to pass WCAG AA at the very least

**Comment by notlee** [comment]
https://github.com/Financial-Times/o-share/pull/134
